docs/CHANGELOG.rst

```diff
@@ -32,6 +32,8 @@ Changed
 - **BACKWARD INCOMPATIBLE:** Remove filtering of genes with low expression
   in PCA analysis
 - Expand error messages in ``rose2`` process
+- Check for errors during download of FASTQ files in import SRA process,
+  upgrade ``fastq-dump`` tool to version 2.9.0
```

Comment:
It might be better to reference specific docker image version, instead of the version of the tool. One can easily identify tool version from the docker version, but not vice-versa.